gemfire/spring-cloud-starter-stream-source-gemfire-cq/src/main/java/org/springframework/cloud/stream/app/gemfire/cq/source/GemfireCqSourceProperties.java

```diff
@@ -25,7 +25,7 @@
  *
  * @author David Turanski
  */
-@ConfigurationProperties
+@ConfigurationProperties("gemfire.cq")
```

Comment:
I think this is fine to be as just `gemfire` as well.
It's enough odd to have to `source` modules in the same stream or even aggregate app.
Or am I missing anything in this prefixes woods?